Kernel/PolynomialNormalizer.cpp

```diff
@@ -192,6 +192,7 @@ Option<NormalizationResult> normalizeSpecializedFractional(Interpretation i, Nor
 {
   switch (i) {
     case NumTraits::divI:
```

Comment:
In these cases `ts` might be `nullptr` in a convoluted way if there are no sub-terms. Assert that it is not when we expect subterms.